test/test_sgg.py

```diff
@@ -115,6 +121,22 @@ def test_seasons():
     assert seasons['data'] is not None
     assert len(seasons['data']) > 0
 
+def test_special():
+    username = 'mfutch78@gmail.com'
+    password = cloud_utils.access_secret_version("593748364912", "FOREUP_MFUTCH", "latest")
+    token = foreup_utils.get_token(username, password)
+    special = foreup_utils.get_special(token, SPECIAL_TEST_CASE['COURSE_ID'], SPECIAL_TEST_CASE['TEESHEET_ID'], SPECIAL_TEST_CASE['SPECIAL_ID'])
```

Comment:
If this special gets deleted in the future, this test case may start to fail but its good to have something in there now to start with.